packages/test/test-end-to-end-tests/src/test/compression.spec.ts

```diff
@@ -85,13 +96,65 @@ const compressionSuite = (getProvider) => {
 			if (provider.type === "TestObjectProviderWithVersionedLoad") {
 				this.skip();
 			}
+			await setupContainers();
 			const value = generateRandomStringOfSize(5);
 			localMap.set("testKey", value);
 
 			await provider.ensureSynchronized();
 			assert.strictEqual(localMap.get("testKey"), value);
 			assert.strictEqual(remoteMap.get("testKey"), value);
 		});
+
+		const messageGenerationOptions = generatePairwiseOptions<{
```

Comment:
I know over time the _order_ these different operations happened in has fluctuated as well.  What test coverage do we have/need for different orders of operations?

Reply:
What order has changed? The only thing I can think of is grouped batching moving to be before compression/chunking, but that happened while it was an experimental feature. So, I don't think we need to test that configuration